doc/sphinx/cti/input-files.rst

```diff
@@ -709,4 +709,3 @@ in the report for Chemkin referenced above. These errors include:
 
 .. [SAND98] See R. J. Kee, G. Dixon-Lewis, J. Warnatz, M. E. Coltrin, J. A. Miller,
    H. K. Moffat, Sandia National Laboratories Report SAND86-8246B (1998).
-   
```

Comment:
@speth This is the section @EmilAtz is referring to: https://github.com/Cantera/cantera/blob/master/doc/sphinx/cti/input-files.rst#syntax-errors I'm not sure if `importPhase` is still used in C++, and also, what language this Traceback is generated by (i.e., if calling from C++ would produce a similar looking traceback). I also think the `Cantera Error` section is out of date.